spec/models/ability_spec.rb

```diff
@@ -250,4 +249,52 @@
       include_examples "a single player event"
     end
   end
+
```

Comment:
You can put all the shared code of the tests in a "before" block. That safes a lot of space.

Reply:
line 72 - 156